__tests__/app.test.js

```diff
@@ -110,3 +111,26 @@ describe('GET /api', () => {
         })
     })
 })
+
+describe('GET /api/articles/:article_id', () => {
+  it('responds with an article object with correct article_id given', () => {
+    return request(app)
+    .get('/api/articles/1')
+    .expect(200)
+    .then((res) => {
+       
+        expect(Object.values(res.body.article).length).toBe(8)
+        expect(res.body.article.article_id).toBe(1)
```

Comment:
I see what you are doing and I respect the logic, however there are better ways of achieving this. Keep an eye on the jest docs for matchers that might be more appropriate. If you know the exact article that is being returned how about explicitly checking for it?